ui/addhardware.ui

```diff
@@ -1,54 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.38.2 -->
+<!-- Generated with glade 3.22.2 -->
```

Comment:
These diffs are huge because you are using an old version of glade, 3.22 was ~4.5 years ago. You need to find something newer.